tmt/checks/avc.py

```diff
@@ -64,6 +66,14 @@ def normalize(
 #: part of the report.
 INTERESTING_PACKAGES = ['audit', 'selinux-policy']
 
+#: Default message types to be collected by ausearch
+DEFAULT_MESSAGE_TYPES = ['AVC', 'USER_AVC', 'SELINUX_ERR']
+
+#: Compiled regex pattern to match relevant AVC denial messages
+DENIAL_PATTERN = re.compile(
+    r'^type=(?:' + '|'.join(map(re.escape, DEFAULT_MESSAGE_TYPES)) + r')\b'
```

Comment:
```suggestion
    rf"^type=(?:{'|'.join(map(re.escape, DEFAULT_MESSAGE_TYPES))})\b"
```
Another way to write this. We don't seem to have a precedence for either format, so either approach is up for grabs.